community/swaylock/checksums

```diff
@@ -0,0 +1 @@
+37b9c70e5698e8872b0b964a7454ecdb0f1a9e06b686eecf2b6c54f7f64a3f42  1.5.tar.gz
```

Comment:
What version of `kiss` are you using?

Reply:
The latest. I copied both sway packages from https://github.com/himmalerin/kiss-wayland and only made a few build changes. Swaylock wasn't updated since that so I didn't have to run the checksums again.

Will fix.